samwise.js

```diff
@@ -8,24 +8,26 @@
 
```

Comment:
SCRIPT-Well done with the efficient error handling, it covers all the possible scenarios.